### DISCUSSION TITLE
MTS videos no EXIF data

### DISCUSSION THREAD

**Discussion by Roadrunner67** [immich-app/immich#8161] [Q&A]
I have some (older) videos in my External Library in the MTS format.
Either there is no EXIF info in the vids, or Immich is unable to deduct them, They all appear in the timeline on the same day, probably file-time. They do show the first frame as a thumbnail, and they play with no issues.
Until now I have found no tagging software that supports MTS videos.
I have tried converting the videos using VLC to MP4 format (reducing the size considerably), and then I can add EXIF information, which places the videos correctly in the timeline, but now they have no Thumbnails. 

Does anyone have tips & tricks on how to deal with this?

**Comment by Hely0n** [comment]
I had the same problem and your best bed is to try "exiftool -ee" (extract embedded) to get some metadata that's baked into the video stream. But when it doesn't, you have to estimate them unfortunately. 

Additionally, MTS videos will be transcoded by immich anyway, so you did it the right way by converting them, but it's important that the videocodec is H.264 and nothing exotic, or it will still be transcoded.

That no thumbnails are shown, seems like a bug, maybe they will come after server reboot, etc.

**Comment by Roadrunner67** [comment]
I had a lot of media files, where exif tags were misinterpreted, but I had the Google Photos sidecar json files, which Immich sadly doesn't support. 
So I sat myself down and built a dotnet app to load the sidecar json meta data and apply them to the original media, one scheme for MTS files and a different scheme for jpg, that's all I need at this point in time.

**Reply by mmomjian** [reply]
What command did you use to convert to MP4? I wouldn't expect the filesize to considerably change and you may have reencoded the video, which is not ideal.  Are they not underlying H264/AVC? Usually you can convert using the following command: `ffmpeg -i x.mts -c copy -movflags faststart out.mp4`

**Reply by Roadrunner67** [reply]
I used VLC on Windows, but I have reverted back to the original MTS videos and updated the EXIF tags
